packages/client/src/store/modules/agencies.js

```diff
@@ -14,10 +14,10 @@ export default {
   },
   actions: {
     fetchAgencies({ commit }) {
-      fetchApi.get('/api/agencies').then((data) => commit('SET_AGENCIES', data));
+      fetchApi.get('/api/organizations/:organizationId/agencies').then((data) => commit('SET_AGENCIES', data));
```

Comment:
at first i thought of adding the organizations path in the fetchApi functions instead of hardcoding it in the modules but there is a route that is agnostic to organization (/sessions) and there might be others in the future.